mm2src/coins/Cargo.toml

```diff
@@ -126,7 +126,7 @@ rust-ini = { version = "0.13" }
 rustls = { version = "0.20", features = ["dangerous_configuration"] }
 secp256k1v22 = { version = "0.22", package = "secp256k1" }
 tendermint-config = { version = "0.23.7", default-features = false }
-tokio = { version = "1.7" }
+tokio = { version = "1.20" }
```

Comment:
`tokio` is changed from `1.7` to `1.2`, why downgrade and not upgrade?

Reply:
It's not downgrade to `1.2`, it's upgrade to `1.20` from `1.7`(13 levels above at minor level).

Reply:
ooh, I confuse this sometimes :). Thanks for the explanation.